### PR TITLE
[TEST] [E2E] Adding e2e tests for readonly access to apps

### PR DESCRIPTION
**Pull request by yellowee**
- Changing changeset to patch
- Original PR: https://github.com/saleor/saleor-dashboard/pull/4747

## What type of PR is this?
- [ ] 💅 Refactor
- [ ] 🌟 Feature
- [ ] 🔥 Bug Fix
- [ ] 🔩 Maintenance
- [ ] 🛠 Workflow CI/CD changes

## Related Issues or Documents

- closes #

## Have you written tests?
- [ ] Yes!
- [ ] No... here is why: _Writing tests are mandatory, please replace this text with why test are not included in this PR_

### CODE REVIEW

**Review by poulch — DISMISSED** [review]

File: playwright/tests/apps.spec.ts [hunk 1]
```diff
@@ -1,53 +1,19 @@
...
-import { AppPage } from "@pages/appPageThirdparty";
-import { APPS } from "@data/e2eTestData";
 
-test.use({ storageState: "./playwright/.auth/admin.json" });
-let appsPage: AppsPage;
-let installationPage: AppInstallationPage;
-let appPage: AppPage;
+test.use({ storageState: "playwright/.auth/app.json" });
 
-test("TC: SALEOR_119 User should be able to install and configure app from manifest @e2e", async ({
```
Comment (szczecha) [inline playwright/tests/apps.spec.ts:12]: Why did you remove this test? It's important to have this covered since we had incident in past.

Comment (yellowee) [reply playwright/tests/apps.spec.ts]: Good catch - I've reverted the unintended changes @szczecha

**Review by szczecha — DISMISSED** [review]

File: src/apps/components/AppDetailsPage/AppAdditionalInfo/index.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export { AppAdditionalInfo } from "./AppAdditionalInfo";
```
Comment (krzysztofzuraw) [inline src/apps/components/AppDetailsPage/AppAdditionalInfo/index.ts:1]: **Question:** why we exported this file? 🤔 

I mean it was causing PW to fail? 🤔

Comment (yellowee) [reply src/apps/components/AppDetailsPage/AppAdditionalInfo/index.ts]: @krzysztofzuraw fixed; I've added this by accident - thank you for pointing it out!

**Review by krzysztofzuraw — DISMISSED** [review]
